develop/package_basics.Rmd

```diff
@@ -0,0 +1,182 @@
+---
+title: "Creating a new R package for population genetics"
+---
+
+Motivation
+=========
+
```

Comment:
It looks you are using both atx style and setext-style headers here. Is this on purpose? Any reason not to just standardize on atx-style? (And do we not say something about that in the guidelines? If not, perhaps we should.)